swan-lake/integration/get-started.md

```diff
@@ -126,13 +133,12 @@ service / on httpListener {
 }
 ```
 
-Let's take a moment to digest the new constructs in this code:
+Let's take a moment to understand the new constructs in this code:
 
 - The <a href="https://lib.ballerina.io/ballerina/http/latest" target="_blank">`http`</a> module provides high-level abstractions to work with the HTTP protocol. 
-- The listener declaration creates a new HTTP listener with the port `8080`. The `listener` is the entity, which receives the network input and then routes it to the attached service(s).
-- The service declaration specifies the listener to which the service gets attached and a collection of remotely accessible methods. There are two kinds of methods as `resource` and `remote`.
-- Services use `remote` methods to expose services in a procedural style and they are named by verbs whereas `resource` methods are used for data-oriented protocols and they are named by nouns.
-- In this example, there are two `resource` methods: The first one responds to HTTP GET requests with the `/greeting` path and the other one responds to `GET` requests with the `/greeting/{name}` path.
+- The listener declaration creates a new HTTP listener that will listen on port `8080`. The `listener` is the entity which receives network input and routes it to the attached service(s).
+- The service declaration specifies the listener to which the service gets attached and a collection of remotely accessible methods. There are two kinds of methods: `resource` methods and `remote` methods. Services use `remote` methods to expose services in a procedural style and they are named using verbs, whereas `resource` methods are used for data-oriented protocols and they are named using nouns.
```

Comment:
For better readability,
```suggestion
- The service declaration specifies the listener to which the service gets attached and a collection of remotely accessible methods. There are two kinds of methods: `resource` methods and `remote` methods. Services use `remote` methods to expose services in a procedural style, and they are named using verbs, whereas `resource` methods are used for data-oriented protocols, and they are named using nouns.
```

Reply:
Will change to

```suggestion
- The service declaration specifies the listener to which the service gets attached and a collection of remotely accessible methods. There are two kinds of methods: `resource` methods and `remote` methods. Services use `remote` methods to expose services in a procedural style and are named using verbs, whereas `resource` methods are used for data-oriented protocols and are named using nouns.
```

(to minimize breaks/commas).